examples/interaction/tools/tool_context_menu.py

```diff
@@ -0,0 +1,49 @@
+""" Example showing how to configure the context menu of a plot, including
```

Comment:
I would expect all of the menu to be shown if possible.

![image](https://github.com/user-attachments/assets/d438db53-e9b7-4427-9f09-3ee4f0a3626b)

Reply:
Started issue https://github.com/bokeh/bokeh/issues/14245 for this.